### PR TITLE
PMP output version update and add new menu (sea-ice and QBO)

### PR DESCRIPTION
**Pull request by lee1043**

### CODE REVIEW

File: research/metrics/v1.7.0/index.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+---
+
+Questions or comments about the PCMDI Simulation Summaries should be sent to the [PMP team](mailto:pcmdi-metrics@llnl.gov).
+
+The efforts of the authors are supported by the [Regional and Global Climate Modeling Program][RGMA] of the [United States Department of Energy's Office of Science][DOEOS]. This work is funded by the Climate and Environmental Sciences Division of the DOE Office of Science and is performed under the auspices of the U.S. Department of Energy by [Lawrence Livermore National Laboratory][LLNL] under contract DE-AC52-07NA27344. LLNL-WEB-812310
+
+[latest]: {{site.baseurl}}/research/metrics/
+
+[v1.7.2]: {{site.baseurl}}/research/metrics/v1.6.0
```
Comment (mauzey1) [inline research/metrics/v1.7.0/index.md:107]:
```suggestion
[v1.7.0]: {{site.baseurl}}/research/metrics/v1.7.0
```

File: research/metrics/v1.7.0/index.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+[metrics]:{{site.baseurl}}/research/metrics/
+
+[mean_clim]: mean_clim
+[enso]: enso
+[variability_modes]: variability_modes
+[mjo]: mjo
+[monsoon]: monsoon
+[precip]: precip
+[sea_ice]: sea_ice
+[qbo]: qbo
```
Comment (mauzey1) [inline research/metrics/v1.7.0/index.md:151]: You will need to make pointers for these directories like for mean_clim and mjo.

**Review by mauzey1 — CHANGES_REQUESTED** [review]

File: research/metrics/v1.7.0/index.md (hunk 2 above)
Comment (lee1043) [reply research/metrics/v1.7.0/index.md:151]: @mauzey1 thank you for reviewing. Pointers are now added via https://github.com/PCMDI/pcmdi.github.io/pull/558/commits/62aea32b0d743c1942a19e62ce4b7162368d05da

File: research/metrics/v1.7.0/qbo/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+###### [Research][research] > [Metrics][metrics] > QBO
+---
+
+# Quasi-Biennial Oscillation (QBO)
+<br/>
+
+
+<br/>
+### Summary statistics in Interactive Bar Charts
+  - (PROTOTYPE) **[QBO-MJO Teleconnection for CMIP5 and CMIP6 Historical][seaice_ibc]**
```
Comment (mauzey1) [inline research/metrics/v1.7.0/qbo/index.md:14]:
```suggestion
  - (PROTOTYPE) **[QBO-MJO Teleconnection for CMIP5 and CMIP6 Historical][qbo-mjo_ibc]**
```

**Review by mauzey1 — CHANGES_REQUESTED** [review]

**Review by mauzey1 — APPROVED** [review]

**Comment by lee1043** [comment]
@mauzey1 thank you for your very helpful review! I appreciate it.

**Pull request merged**